service/src/main/java/uk/gov/hmcts/reform/fpl/validation/validators/time/TimeRangeValidator.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.validation.validators.time;
+
+import uk.gov.hmcts.reform.fpl.validation.interfaces.time.TimeDifference;
+import uk.gov.hmcts.reform.fpl.validation.interfaces.time.TimeRange;
+
+import java.time.LocalDateTime;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class TimeRangeValidator implements ConstraintValidator<TimeRange, LocalDateTime> {
+
+    private TimeDifference before;
+    private TimeDifference after;
+    private boolean inclusive;
+
+    @Override
+    public void initialize(TimeRange annotation) {
+        before = annotation.rangeBefore();
+        after = annotation.rangeAfter();
+        inclusive = annotation.inclusive();
+    }
+
+    @Override
+    public boolean isValid(LocalDateTime value, ConstraintValidatorContext context) {
+        LocalDateTime now = LocalDateTime.now();
```

Comment:
Again with the time config stuff, although I don't know if it is possible in a validator

Reply:
Maybe annotating the class as a component will allow for it to be autowired nicely (not sure though)

Reply:
Had a chat with Etim and it should be able to be done with the component annotation.

Reply:
Caught up with Etim and he does not think it's possible to use as a JSR validator :/ He suggested we keep as it is.

Reply:
This is very sad

Reply:
no longer very sad